src/components/standalone/InputLayout.vue

```diff
@@ -0,0 +1,26 @@
+<script lang="ts" setup>
+defineProps({
+  title: String,
+  description: String
+})
+
+defineOptions({
```

Comment:
Let's keep the default policy for component attributes; I'd remove `defineOptions`